steps/circuit.py

```diff
@@ -149,3 +149,31 @@ def create_circuit_set_from_circuit_artifacts(
             circuit_set_object.append(load_circuit(object))
 
     save_circuit_set(circuit_set_object, "circuit-set.json")
+
+def create_arbitrary_single_qubit_circuit(rz_parameter_1: float, 
+                                        ry_parameter:float, 
+                                        rz_parameter_2:float
+                                        ):
+
+    circuit = Circuit()
+    circuit.qubits = [Qubit(0)]
+    gates = [Gate('Ry', qubits=[Qubit(0)], params = [ry_parameter_1]), Gate('Rz', qubits=[Qubit(0)], params = [rz_parameter]), Gate('Ry', qubits=[Qubit(0)], params = [ry_parameter_2]) ]
+    circuit.gates = gates
+    save_circuit(circuit, 'circuit.json')
+
+
+def create_two_qubit_molecular_hydrogen_circuit(rz_parameter:float):
```

Comment:
Same questions & suggestions as for  `create_arbitrary_single_qubit_circuit`. Sorry if I'm being a bit meticulous  :)

Reply:
Removed in 98ccc5269e9797856205403d92be039d36a0de38